components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/Main.java

```diff
@@ -7,6 +7,7 @@
 
 public class Main {
     public static void main(String[] args) {
+        System.setProperty("picocli.disable.closures", "true");  // improves startup time
```

Comment:
Typically, sys props are set by the invoker, IMO. Are we certain that the sys prop set here is actually picked up, and not set too late? If so, please merge the PR to main.

Reply:
I went this route so as to avoid having to expose this detail to the code that invokes the script, but it's a good point that maybe it is ineffective here.

@erichaagdev would you be able to test an alternative implementation where the system property is set when the utility is invoked by the scripts (using `-D`)?  Maybe that's why we aren't seeing any kind of performance gain with this change.